10x/count/count.wdl

```diff
@@ -1216,6 +1216,26 @@ task filter_barcodes {
   }
 }
 
+task get_trimmed_and_aligned {
+    Array[File] trimmed_seqs
+    File genome_output
+    Array[Int] gem_groups
+
+    command <<<
+    >>>
+
+    runtime {
+        docker: "marcusczi/cellranger_clean:cromwell"
+        memory: "30 GB"
+        disks: "local-disk 50 HDD"
+    }
+
+    output {
+      Array[Pair[File, File]] trimmed_and_aligned  =  zip(trimmed_seqs, genome_output)
+      Array[Pair[Int, Pair[File, File]]] gem_group_trimmed_and_aligned  =  zip(gem_groups, trimmed_and_aligned)
+    }
+}
+
```

Comment:
Does moving this into a task mean that it's going to start up a new instance?

Reply:
Yes, it will. This is to work around a limitation in Cromwell when trying to run the count wdl in an import. It doesn't like having this zip command on its own outside of a task in that case. I suggested we put it in a task of its own to get around this, but I forgot that this would cause all the files to get copied down to one VM and that they might be big files. In general, will these be large files? If so, we should see if there's any way to work around this issue without having to copy everything to one VM.

Reply:
They could be pretty big since believe this will download all the sequences and all their alignments. So it's probably something we'll have to address in the future. The way the align/extract tasks are parallelized is probably a little inconsistent with how WDL is supposed to work anyway.

What do you mean by "in an import"?

Reply:
So @samanehsan made a wrapper WDL that imports the 10x count WDL and takes a bundle id and version as inputs, talks to blue box to get the actual fastq paths, runs the 10x WDL, then submits the results back to ingest. We'll run it that way for the demo. In theory anything that works in a standalone WDL should also work when importing that WDL into another workflow, but that zip command seems to be a problem when importing only.

Reply:
Ah okay, thanks for the explanation.